src/quo/components/links/internal_link_card/channel/view.cljs

```diff
@@ -0,0 +1,92 @@
+(ns quo.components.links.internal-link-card.channel.view
+  (:require
+    [quo.components.icon :as icon]
+    [quo.components.links.internal-link-card.channel.style :as status-link-previews-style]
+    [quo.components.links.link-preview.style :as style]
+    [quo.components.markdown.text :as text]
+    [quo.theme :as theme]
+    [react-native.core :as rn]))
+
+(defn- description-comp
+  [description]
+  [rn/view {:style {:margin-bottom 4}}
+   [text/text
+    {:size                :paragraph-2
+     :number-of-lines     3
+     :accessibility-label :description} description]])
+
+(defn- title-comp
+  [title channel-name]
+  [rn/view
+   {:style {:flex-direction :row
+            :align-items    :center}}
+   [text/text
+    {:size                :paragraph-1
+     :number-of-lines     1
+     :weight              :semi-bold
+     :style               (dissoc style/title :flex)
+     :accessibility-label :title}
+    title]
+   [icon/icon :i/chevron-right status-link-previews-style/channel-chevron-props]
+   [text/text
+    {:size                :paragraph-1
+     :number-of-lines     1
+     :weight              :semi-bold
+     :style               (dissoc style/title :flex)
+     :accessibility-label :title}
+    channel-name]])
+
+(defn- thumbnail-comp
+  [thumbnail]
+  [rn/image
+   {:style               status-link-previews-style/thumbnail
+    :source              thumbnail
+    :accessibility-label :thumbnail}])
+
+(defn- logo-comp
+  [logo]
+  [rn/image
+   {:accessibility-label :logo
+    :source              (if (string? logo)
+                           {:uri logo}
+                           logo)
+    :style               (assoc style/logo :margin-bottom 2)}])
+
+(defn- loading-view
+  [theme]
+  [rn/view
+   {:accessibility-label :loading
+    :style               {:height 215}}
+   [rn/view {:style {:flex-direction :row}}
+    [rn/view {:style status-link-previews-style/row-spacing}
+     [rn/view (status-link-previews-style/loading-circle theme)]
+     [rn/view (assoc (status-link-previews-style/loading-first-line-bar theme) :margin-right 4)]]
+    [rn/view {:style status-link-previews-style/row-spacing}
+     [rn/view (status-link-previews-style/loading-circle theme)]
+     [rn/view (status-link-previews-style/loading-first-line-bar theme)]]]
+   [rn/view {:style (status-link-previews-style/loading-second-line-bar theme)}]
+   [rn/view {:style (status-link-previews-style/loading-thumbnail-box theme)}]])
+
+(defn internal-view
```

Comment:
I think we should aim to be consistent in the codebase, in pretty much every other use we have it as `view-internal` 👍